### PR TITLE
Updated for Atomix upgrade to v1beta1

### PR DESCRIPTION
**Pull request by SeanCondon**
Depends on
* https://github.com/onosproject/onos-config/pull/1069 for `onos-config` and
* https://github.com/onosproject/onos-topo/pull/86 for `onos-topo`

### CODE REVIEW

File: onos-topo/values.yaml [hunk 1]
```diff
@@ -22,7 +22,10 @@ store:
   controller: atomix-controller.kube-system.svc.cluster.local:5679
   raft:
     partitions: 1
-    partitionSize: 1
+    clusters: 1
+    replicas: 1
+    backend:
+      image: atomix/raft-replica:latest
```
Comment (adibrastegarnia) [inline onos-topo/values.yaml]: Not sure about this image yet. It should be confirmed by Jordan.

Comment (kuujo) [reply onos-topo/values.yaml]: It is correct.

File: onos-config/templates/partitionset.yaml [hunk 2]
```diff
@@ -9,11 +9,12 @@ metadata:
...
   template:
     spec:
-      size: {{ .Values.store.raft.partitionSize }}
-      raft:
-        electionTimeoutMillis: 5000
-        heartbeatPeriodMillis: 1000
-{{- end }}
\ No newline at end of file
+      partitions: {{ .Values.store.raft.partitions }}
+      replicas: {{ .Values.store.raft.replicas }}
```
Comment (kuujo) [inline onos-config/templates/partitionset.yaml]: `replicas` is a property of the `backend` object below and should be moved there

File: onos-topo/templates/partitionset.yaml [hunk 3]
```diff
@@ -9,11 +9,12 @@ metadata:
...
   template:
     spec:
-      size: {{ .Values.store.raft.partitionSize }}
-      raft:
-        electionTimeoutMillis: 5000
-        heartbeatPeriodMillis: 1000
-{{- end }}
\ No newline at end of file
+      partitions: {{ .Values.store.raft.partitions }}
+      replicas: {{ .Values.store.raft.replicas }}
```
Comment (kuujo) [inline onos-topo/templates/partitionset.yaml]: ditto

**Comment by kuujo** [comment]
Also should change `partitionset.yaml` filenames to `database.yaml`

**Pull request merged**